authguard.service

```diff
@@ -11,11 +11,11 @@ Restart=always
 # optional systemd hardening-options
 # Documentation about these can be found under systemd.exec(5)
 # The linked websites might be for a newer systemd version than your installed one
-#NoNewPrivileges=true     # https://www.freedesktop.org/software/systemd/man/systemd.exec.html#NoNewPrivileges=
-#PrivateDevices=true       # https://www.freedesktop.org/software/systemd/man/systemd.exec.html#PrivateDevices=
-#ProtectSystem=full       # https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectSystem=
 #CapabilityBoundingSet=   # https://www.freedesktop.org/software/systemd/man/systemd.exec.html#CapabilityBoundingSet=
 #ProtectHome=true         # https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectHome=
+#ProtectSystem=full       # https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectSystem=
+#PrivateDevices=true      # https://www.freedesktop.org/software/systemd/man/systemd.exec.html#PrivateDevices=
+#NoNewPrivileges=true     # https://www.freedesktop.org/software/systemd/man/systemd.exec.html#NoNewPrivileges=
```

Comment:
May I add, that this is completely out of order now if you compare it to the man page?